huxley/www/static/js/huxley/components/AdvisorProfileView.js

```diff
@@ -61,169 +62,171 @@ var AdvisorProfileView = React.createClass({
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Number of Spanish Speaking
-                Delegates</td>
-                <td className="field">
-                  {school.spanish_speaking_delegates}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Number of Chinese Speaking
-                Delegates</td>
-                <td className="field">
-                  {school.chinese_speaking_delegates}
-                </td>
-              </tr>
-              <tr>
-                <th colSpan="2">Primary Contact Information</th>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Name</td>
-                <td className="field">
-                  {school.primary_name}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Email</td>
-                <td className="field">
-                  {school.primary_email}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Phone</td>
-                <td className="field">
-                  {school.primary_phone}
-                </td>
-              </tr>
-              <tr>
-                <th colSpan="2">Secondary Contact Information</th>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Name</td>
-                <td className="field">
-                  {school.secondary_name}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Email</td>
-                <td className="field">
-                  {school.secondary_email}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Phone</td>
-                <td className="field">
-                  {school.secondary_phone}
-                </td>
-              </tr>
-              <tr>
-                <th>Fees</th>
-                <td className="field">
-                  <a className="button button-green" href={invoiceUrl} target="_blank">
-                    Generate Your Invoice
-                  </a>
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Fees Owed</td>
-                <td className="field">
-                  {'$' + school.fees_owed.toFixed(2)}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Fees Paid</td>
-                <td className="field">
-                  {'$' + school.fees_paid.toFixed(2)}
-                </td>
-              </tr>
-              <tr>
-                <td className="fieldLabel">Balance</td>
-                <td className="field">
-                  {'$' + (school.fees_owed - school.fees_paid).toFixed(2)}
-                </td>
-              </tr>
+              <tbody>
```

Comment:
Same as thing, should we be using `<thead>` here and then `<tbody>` for everything else? Or does that affect the appearance too adversely?